### ISSUE TITLE
Mac agent offline after reboot/shutdown

### ISSUE THREAD

**Issue by trenton92** [Ylianst/MeshCentral#1420]
Hi. After reboot, the agent does not start. Launchctl restart still helps. I saw that someone had the same issue in Issue. MeshCentral - 0.5.45. If someone found a solution or wrote a script to reconnect the agent, then please help.

Feedback:
KVM in Mac OS works very well. Thank you very much for your efforts.

**Comment by trenton92** [comment]
I wrote my Daemon, put it in LaunchDaemon.
It runs a shell script:
launchctl stop meshagent
launchctl start meshagent

I do the launchctl unload -w / path, then
launchctl load -w / path.
The script is triggered and the agent appears on the network.
After the reboot, the agent does not rise, the same actions help.

Perhaps the problem about which I wrote above just lies not in the meshcentral code, but in the p.list.

Sorry if I write nonsense, I'm very bad on mac os
